python/nano/src/bigdl/nano/utils/common/checker.py

```diff
@@ -16,6 +16,11 @@
 
 from importlib.util import find_spec
 from bigdl.nano.utils.common import CPUInfo
+import os
+import operator
+import subprocess
+from bigdl.nano.utils.common import invalidInputError
```

Comment:
why we need `os`, `subprocess` and `invalidInputError`

Reply:
Just forget to delete them ：） Have fixed.